app.js

```diff
@@ -1,14 +1,14 @@
 // import functions and grab DOM elements
-import { rawPokemonData } from './pokemon.js';
+import { pokemonData } from './pokemon.js';
+import { findById, setInLocalStorage } from './utils.js';
 
 const radios = document.querySelectorAll('input');
-const next = document.querySelector('button');
-const capture = document.querySelector('#capture');
 const images = document.querySelectorAll('label > img');
 
 // initialize state
-let pokemonCaptured = 0;
-let pokemonEncountered = 0;
+let totalRounds = 0;
+const resultsArray = [];
+const POKEMON = 'POKEMON';
```

Comment:
nice work transferring over your work to localstorage.  I realize that you did this in other projects, but definitely did a wonderful job implementing that logic here.